### PR TITLE
corebird: fix cross compiling

### PR DESCRIPTION
**Pull request by Sir-Boops**
* Tried on armv7l-musl and it seems to work fine ( Only launched it, i didn't go any deeper )

### CODE REVIEW

File: srcpkgs/corebird/template [hunk 1]
```diff
@@ -14,10 +14,13 @@ hostmakedepends="pkg-config intltool vala glib-devel gdk-pixbuf libgee08-devel
...
-	arm*) broken="https://build.voidlinux.eu/builders/armv7l-musl_builder/builds/12500" ;;
-esac
+if [ -n "${CROSS_BUILD}" ]; then
+	hostmakedepends+=" gspell-devel"
+fi
 
 pre_configure() {
 	autoreconf -fi
+	if [ -n "${CROSS_BUILD}" ]; then
+		sed -i '13080,13083d' configure
```
Comment (ebfe) [inline srcpkgs/corebird/template]: Please use a pattern that has a chance to survive an update.

Comment (ebfe) [inline srcpkgs/corebird/template]: Even better: drop this and `hostmakedepends+=gstreamer1-devel"

Comment (Sir-Boops) [inline srcpkgs/corebird/template]: Wow, ya that's alot cleaner 😄

File: srcpkgs/corebird/template [hunk 2]
```diff
@@ -14,9 +14,9 @@ hostmakedepends="pkg-config intltool vala glib-devel gdk-pixbuf libgee08-devel
 makedepends="gtk+3-devel rest-devel libgee08-devel sqlite-devel gspell-devel
  libsoup-devel json-glib-devel gettext-devel gst-plugins-base1-devel"
 
-case "$XBPS_TARGET_MACHINE" in
-	arm*) broken="https://build.voidlinux.eu/builders/armv7l-musl_builder/builds/12500" ;;
-esac
+if [ -n "${CROSS_BUILD}" ]; then
+	hostmakedepends+=" gspell-devel gstreamer1-devel"
```
Comment (asergi) [inline srcpkgs/corebird/template]: Just add both to hostmakedepends directly and then you can remove  the if statement at all.

Comment (Sir-Boops) [inline srcpkgs/corebird/template]: Ok, I was under the impression that this was the way you should add arch specific dependencies

**Pull request merged**